src/database/migration.rs

```diff
@@ -0,0 +1,65 @@
+use super::database::Database;
+use rusqlite::{params};
+use std::sync::Arc;
+
+pub struct Migration {
+    database: Arc<Database>,
+}
+
+impl Migration {
+    pub fn migrate_data_03_to_04(&self) {
+        self.database
+            .execute_sql(
+                "alter table tcn rename column contact_time to contact_start;",
+                params![],
+            )
+            .unwrap();
+        self.database
+            .execute_sql(
+                "alter table tcn add column contact_end integer not null default 0;",
+                params![],
+            )
+            .unwrap();
+        self.database
+            .execute_sql(
+                "alter table tcn add column min_distance real default 32.0;",
```

Comment:
which default values to use?

Reply:
Anyway, since the solution is already in place, I'd just drop and create the TCN table here.

Reply:
Ok